hack/dind-cluster.sh

```diff
@@ -160,6 +160,59 @@ function stop() {
   done
 }
 
+function refresh() {
+  local origin_root=$1
+  local config_root=$2
+  local cluster_id=$3
+  local wait_for_cluster=$4
+
+  echo "Refreshing dind cluster '${cluster_id}'"
+
+  # Stop the master and node openshift processes
+  echo "  Stopping master and node processes in cluster '${cluster_id}'..."
+  local master_cid
+  master_cid="$(${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}")"
+  if [[ "${master_cid}" ]]; then
+    ${DOCKER_CMD} exec "${master_cid}" systemctl stop openshift-master.service > /dev/null
+    ${DOCKER_CMD} exec "${master_cid}" systemctl stop openshift-node.service > /dev/null
+  fi
+
+  local node_cids node_count
+  node_cids="$(${DOCKER_CMD} ps -qa --filter "name=${NODE_PREFIX}")"
+  node_count=0
+  if [[ "${node_cids}" ]]; then
+    node_cids=(${node_cids//\n/ })
+    for cid in "${node_cids[@]}"; do
```

Comment:
does 

```sh
for cid in $( ${DOCKER_CMD} ps -qa --filter "name=${NODE_PREFIX}" ); do

done
```

get you what you want? it will break on spaces and newlines by default